cmd_unix.go

```diff
@@ -0,0 +1,39 @@
+// +build darwin dragonfly freebsd linux nacl netbsd openbsd solaris
```

Comment:
Should we just make this `// +build !windows` ?

Reply:
I'm not really sure what the best thing to do is. I guess hosts-files are not used on Native Client (nacl), and on Plan 9 they are placed under `/lib/ndb/hosts` (according to [Wikipedia](https://en.wikipedia.org/wiki/Hosts_(file)#Location_in_the_file_system)).

Perhaps the best thing is to only build for targets that have been tested? Although things should at least work fine on the BSDs, I think...

Reply:
Oh, interesting. Um, we could exit with an error on those platforms we are not sure about? Or make our best guess.

Reply:
Perhaps just keep this file and add `cmd_plan9.go`? On NaCl it'll probably exit with a "file not found" by default, which I guess is as good an error message as any (and no more work is needed)...

Just make it clear in the readme that the tool has only been tested on Linux, macOS and Windows.

Reply:
Yeah that sounds fine to me, I can add the README text.